Source/Core/Core/Boot/Boot.cpp

```diff
@@ -388,6 +388,11 @@ bool CBoot::BootUp()
       PowerPC::DBATUpdated();
       PowerPC::IBATUpdated();
 
+      // Because there is no TMD to get the requested system (IOS) version from,
+      // we default to IOS58, which is the version used by the Homebrew Channel.
+      if (dolLoader.IsWii())
+        SetupWiiMemory(0x000000010000003a);
```

Comment:
Well, I don't know, booto was the one who added the SetupWiiMemory call for ELFs. I think it'd make more sense for the IOS version to be 58, because 80 is only supposed to be used by the system menu. Should I change it in this PR though?